### ISSUE TITLE
sync drm/pvr with newer codebase

### ISSUE THREAD

**Issue by thopiekar** [EMGD-Community/intel-binaries-linux#23]
probably a solution fixing a lot of problems with our code or might break everything :/

https://git.ti.com/graphics/omap5-sgx-ddk-linux

**Comment by mirh** [comment]
I guess like you can check the repos I linked [here](https://github.com/EMGD-Community/intel-binaries-linux/commit/7afa9a5322724e12eff01fb7d08ca53f5da94f97#commitcomment-26180509) for a follow up on that. 
From what I understood.. you can even patch the DDX driver to support newer Xorg?

And latest development is at: https://github.com/openpvrsgx-devgroup/linux_openpvrsgx/wiki

**Comment by thopiekar** [comment]
Nope, as far as I understand the Xorg driver is always a blob on the user-space level. However, the driver (DRM) is provided as source.
Your repo might contain improvements for the copy of the DRM driver here (as EMGD seems to depend on the sgx driver), but disposed my eeepc already as the battery was blowing up.

**Comment by mirh** [comment]
Mhh what? I thought the DDX had always been [open](https://github.com/lkundrak/xserver-xorg-video-psb)?
In fact there even seem to be [another one](https://github.com/patjak/xf86-video-gma500).

**Comment by mirh** [comment]
And there you are indeed
https://github.com/EMGD-Community/intel-binaries-linux/issues/58

**Comment by mirh** [comment]
@goldelico you have a far newer codebase than 3.7 here

**Comment by goldelico** [comment]
Nice! I'll collect (let's open a museum :) and compare. Thanks.

**Comment by goldelico** [comment]
Well, it may be maintained/adapted for newer kernel but it is an even older version of the DDK:
#define PVRVERSION_STRING "1.5.15.3226"
